src/binary-search-tree.js

```diff
@@ -5,6 +5,43 @@ class BinarySearchTree {
     this.left = null;
     this.right = null;
   }
+  insert(stuff) {
+    const node = new BinarySearchTree(stuff);
+    function recurse(bst) {
+      if (stuff < bst.value && bst.left === null) {
+        bst.left = node;
+      } else if (stuff < bst.value) {
+        recurse(bst.left);
+      } else if (stuff > bst.value && bst.right === null) {
+        bst.right = node;
+      } else if (stuff > bst.value) {
+        recurse(bst.right);
+      }
+    }
+    recurse(this);
+  }
+  contains(value) {
```

Comment:
Here, you can streamline your implementation by just doing 
```
if (value === this.value) return true;
else if (value > this.value && this.right) return this.right.contains(value);
else if (value < this.value && this.left) return this.left.contains(value);
else return false;
```
You don't need to perform the explicit checks for `this.right === null` and `this.left === null`.